### PR TITLE
(modules): Add Advice module

### PR DESCRIPTION
**Pull request by milouk**
This PR introduces a new module to Olivia. Now Olivia can support giving random advices.

advice

### CODE REVIEW

**Comment by hugolgst** [comment]
I like that but the API is giving advice only in English, do you think that it is a problem?

**Comment by milouk** [comment]
ehm honestly I thought about it, but on the other hand almost all APIs hold their info in english. Take the Jokes API, for example. If we consider this to be an issue then we are very limited in terms of APIs. As the project grows and more languages get added, the harder it will be for us to adapt and find new multilingual APIs. I think its a compromise we will have to make.

**Comment by hugolgst** [comment]
You're right

**Comment by hugolgst** [comment]
I'll try to review this PR tomorrow, thanks a lot :)

File: modules/advices.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package modules
+
+import (
+	"fmt"
+	"net/http"
+	"io/ioutil"
+	"github.com/tidwall/gjson"
```
Comment (hugolgst) [inline modules/advices.go:7]: I think the usage of this library is useless, you can just unmarshal the content.

Comment (milouk) [reply modules/advices.go:7]: I used this lib to avoid having a nested struct. If you are ok with that I have no problem removing it.

Comment (hugolgst) [reply modules/advices.go:7]: You can do that without a structure. Just create a `map[string]interface{}` variable and unmarshal the content inside it.
Then you can access elements by doing `myVar["field"]`

**Comment by milouk** [comment]
Hey @hugolgst is there anything else needed or something I didn't address?

**Comment by hugolgst** [comment]
Sorry for that I was pretty busy

**Pull request merged**